aggsender/types/status.go

```diff
@@ -22,11 +22,11 @@ type AggsenderStatus struct {
 }
 
 type AggsenderInfo struct {
-	AggsenderStatus          AggsenderStatus `json:"aggsender_status"`
-	Version                  zkevm.FullVersion
-	EpochNotifierDescription string        `json:"epoch_notifier_description"`
-	NetworkID                uint32        `json:"network_id"`
-	Mode                     AggsenderMode `json:"mode"`
+	AggsenderStatus AggsenderStatus `json:"aggsender_status"`
+	Version         zkevm.FullVersion
```

Comment:
Nitpick, I'd suggest to change the import alias

```suggestion
	Version         aggkit.FullVersion
```

Reply:
Done: https://github.com/agglayer/aggkit/pull/1237/commits/6b83f6bb2dadac3cb3fac8dfeef7a588badfd7e8